src/features/flows/nRF52/index.ts

```diff
@@ -63,7 +63,8 @@ const programConfig = [
     {
         name: 'Peripheral UART Service',
         type: 'jlink',
-        description: 'Check that UART over Bluetooth® LE is working.',
+        description:
+            'This sample demonstrates how to use the Nordic UART Service (NUS).',
```

Comment:
My thoughts evolved since I sent my mail in reply to Ingar's thread. Let's not promise working with BLE off the bat, but prepare the user for what he can do later:
```suggestion
            'Program the Nordic UART Service (NUS) to test it later with Bluetooth(r) LE.',
```